source/keyboardHandler.py

```diff
@@ -35,13 +35,16 @@
 import NVDAState
 from contextlib import contextmanager
 import threading
+import winKernel
 
 if typing.TYPE_CHECKING:
 	from NVDAObjects import NVDAObject  # noqa: F401
 	from watchdog import WatchdogObserver
 
 _watchdogObserver: typing.Optional["WatchdogObserver"] = None
 ignoreInjected=False
+_lastInjectedKeyUp: typing.Tuple[int, int] = None

```

Comment:
I believe this is the standard syntax for python 3.11 now

```suggestion
_lastInjectedKeyUp: Optional[tuple[int, int]] = None
```

Reply:
alternatively

```suggestion
_lastInjectedKeyUp: tuple[int, int] | None = None
```